src/client/websocket/packets/handlers/GuildMembersChunk.js

```diff
@@ -10,10 +9,9 @@ class GuildMembersChunkHandler extends AbstractHandler {
     const guild = client.guilds.get(data.guild_id);
     if (!guild) return;
 
-    const members = data.members.map(member => guild._addMember(member, false));
+    const members = new Collection(data.members.map(member => [member.user.id, guild._addMember(member, false)]));
```

Comment:
Create collection, loop + add. Don't abuse the array shortcut methods, since you'll be creating a bunch of trash arrays that will just have to be cleaned up by the GC.